docs/release_process.md

```diff
@@ -0,0 +1,36 @@
+Unless otherwise noted, follow the guidance here:
+https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance
+
+The most important rules are:
+
+1. Never publish anything that is not committed.
+2. Never commit anything that hasn't been reviewed and approved.
+
+It's critical to keep in mind that in the Dart package ecosystem, publishing is
+_forever_.
+This means special care has to be taken to make sure that everything is correct
+before a package is published. The Dart team has spent hours and in some cases
+days fixing problems due to a mistake when publishing.
+It is hard to be _too_ careful here.
+
+Assuming that we want to release both
+[`functions_framework`](https://pub.dev/packages/functions_framework) and
+[`functions_framework_builder`](https://pub.dev/packages/functions_framework_builder)
+at the same time, follow these steps.
+
+1. Prepare a PR to release functions_framework.
+   - Remove the `-dev` from `pubspec.yaml` and `CHANGELOG.md`.
+2. Get the PR approved and commit it.
```

Comment:
Use "1." Everywhere for ordered lists in markdown

Reply:
ack